host/4/bullseye/dotnet-inproc/dotnet-appservice.Dockerfile

```diff
@@ -57,7 +57,7 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
 RUN apt-get update && \
     apt-get install -y libc-dev
 
-COPY --from=runtime-image [ "/usr/share/dotnet", "/usr/share/dotnet" ]
+COPY --from=runtime-image [ "/usr/share/dotnet/shared", "/usr/share/dotnet/shared" ]
```

Comment:
Doesn't work like this : 

![image](https://github.com/user-attachments/assets/0e2f9e4a-556e-496a-8190-8976a7193063)